v19.2/explain.md

```diff
@@ -396,12 +450,14 @@ If there were an index on `v`, CockroachDB would be able to avoid scanning the e
 ~~~
 
 ~~~
- tree | field | description
-------+-------+-------------
- scan |       |
-      | table | kv@v
-      | spans | /4-/6
-(3 rows)
+  tree |    field    | description
++------+-------------+-------------+
+       | distributed | false
+       | vectorized  | false
+  scan |             |
```

Comment:
What does it mean when both of the above are false?  I think we should EXPLAIN this (lol get it)